definitions.md

```diff
@@ -0,0 +1,50 @@
+# Definition of Common Terms
+
+## Infrastructure
+
+- <a name="hub"/>**Hub**:
+    - **Definition**: A kubernetes cluster that is running OCM, typically connected to one or more [spokes](#spoke).
+    - Common alternatives to avoid:
+        - Control plane
+        - Control cluster
+- <a name="spoke"/>**Spoke**:
+    - **Definition**: A Kubernetes cluster connected to a [hub](#hub) where resources can be placed via a manifestwork.
```

Comment:
not sure I fully agree here a Spoke is referred to as a ManagedCluster even in OCM

```suggestion
    - **Definition**: A Kubernetes cluster connected to and managed by a [hub](#hub) where resources can be placed via a manifestwork.
```